### PR TITLE
Feature/setup-sfpowerkit-noheader-env-var

### PR DESCRIPTION
**Pull request by henry88lay**
Set up an environment variable for sfpowerkit to not display the header and ran tests on commands.

### CODE REVIEW

File: packages/sfpowerscripts-cli/src/SfpowerscriptsCommand.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { SfdxCommand } from "@salesforce/command";
 import { OutputFlags } from "@oclif/parser";
 import SFPStatsSender from "@dxatscale/sfpowerscripts.core/lib/stats/SFPStatsSender";
 import * as rimraf from "rimraf";
+import child_process = require("child_process");
```
Comment (azlam-abdulsalam) [inline packages/sfpowerscripts-cli/src/SfpowerscriptsCommand.ts]: Unused?

Comment (henry88lay) [reply packages/sfpowerscripts-cli/src/SfpowerscriptsCommand.ts]: yeah its unused i am going to remove it

**Comment by azlam-abdulsalam** [comment]
@henry88lay  can you add a condition on the variable, so that header is enabled if the loglevel is set to debug or below? This could be helpful to understand what is the version of sfpowerkit being used

**Comment by henry88lay** [comment]
> @henry88lay can you add a condition on the variable, so that header is enabled if the loglevel is set to debug or below? This could be helpful to understand what is the version of sfpowerkit being used

good idea i will look into getting that implemented.

**Pull request merged**